src/chunker/layout.js

```diff
@@ -467,233 +530,572 @@ class Layout {
 		}
 
 		if (!node) {
+			// console.log(isElement(container), child(container, offset), temp);
 			return;
 		}
 
-		return new BreakToken(
+		return new Overflow(
 			node,
-			offset
+			offset,
+			overflow.getBoundingClientRect().height,
+			overflow
 		);
 
 	}
 
-	findBreakToken(rendered, source, bounds = this.bounds, prevBreakToken, extract = true) {
-		let overflow = this.findOverflow(rendered, bounds);
+	lastChildCheck(parentElement, rootElement) {
+		if (parentElement.childElementCount) {
+			this.lastChildCheck(parentElement.lastElementChild, rootElement);
+		}
+
+		let refId = parentElement.dataset['ref'];
+
+		if (['TR', 'math', 'P'].indexOf(parentElement.tagName) > -1 && parentElement.textContent.trim() == '') {
```

Comment:
Can you add a comment on what this is doing?

Wondering if we need this list to have any other elements and if it is ok to have `math` lowercase.

Reply:
Yeah, the tagname for math was lowercase, which is unusual. Will add a comment and remind myself what's special about TR and P.